Files/Helpers/JumpListManager.cs

```diff
@@ -59,50 +60,70 @@ private void AddFolder(string path)
         {
             if (instance != null && !JumpListItemPaths.Contains(path))
```

Comment:
Forgot this one: it should be only `if (instance != null)` so if you visit a path again it will be moved to the top of the list (it will still appear only once).